src/vivarium_cluster_tools/vipin/log_parser.py

```diff
@@ -0,0 +1,91 @@
+from loguru import logger
```

Comment:
The general convention for sorting python imports is to do them in three blocks:
```
# Stdlib imports
import json
from pathlib import Path
import re
from typing import Union

# Single line of whitespace, then external libs
from loguru import logger
import numpy as np
import pandas as pd
from pandas.io.json import json_normalize
import requests

# Another newline then internal imports
from vivarium_cluster_tools.vipin import utilities  # E.g
```

There are tools we can put in our CI pipeline to keep this sorted for us.

Reply:
Thanks for this. I'll update for this.